app/services/heroku_commands.rb

```diff
@@ -164,5 +167,10 @@ def remove_accounting_keys
       def write_log(message)
         puts message
       end
+
+      def check_valid_app(app_name)
+        _output, error = Open3.capture3("heroku apps:info #{app_option(app_name)}")
```

Comment:
this is just an easy and safe command used as a preflight check, nothing really special about `apps:info`

Reply:
that is awesome dude!